packages/developer-portal/src/components/apps/pipeline/pipeline-configure/pipeline-configure.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { FC } from 'react'
-import { BodyText } from '@reapit/elements'
+import { BodyText, Subtitle } from '@reapit/elements'
```

Comment:
Don't think `Subtitle` is used? You'll need to delete this for linting

Reply:
Good spot - surprised the PR succeeded actually. Updated in latest commit

Reply:
that's what I thought!